machine_management/control_plane_machine_management/cpmso-getting-started.adoc

```diff
@@ -52,12 +52,18 @@ The status of the control plane machine set after installation depends on your c
 |
 |
 |X
+
+|{rh-openstack-first}
+|X ^[4]^
```

Comment:
Looks like this has the same condition as Nutanix, so let's share a number to avoid a long list here :)
```suggestion
|X ^[3]^
```

Reply:
Okay